src/main/scala/com/cognite/sdk/scala/v1/resources/files.scala

```diff
@@ -36,6 +37,11 @@ class Files[F[_]](val requestSession: RequestSession[F])
         value => value
       )
 
+  override def createItems(items: Items[FileCreate]): F[Seq[File]] = {
+    import cats.implicits._
+    applicative.map(items.items.toList.traverse(createOne))(_.toSeq)
```

Comment:
With the right cats syntax import you should be able to do `items.items.toList.traverse(createOne).map(_.toSeq)`.
`import cats.syntax._`, probably - you can check where it came from in the 3D revisions `.map`.